module-2/data-analysis-with-pandas/your-code/Main.DAwPd.ipynb

```diff
@@ -0,0 +1,2528 @@
+{
```

Comment:
- The first thing you should do when importing your data set is to study the main characteristics of your data set. One of your best friends will be pandas-profiling. Install pandas-profiling, run pip install pandas-profiling on your term, afterwards impor the library on your Jupyter notebook and finally apply the following method to your final dataframe
```Python
pandas_profiling.ProfileReport(df)
```
- When  you need to split a column in two as your datetime column, it's a good idea to use extract method with expand = True. Afterwards, you can convert the columns to the proper type. Extract method is made for this kind of stuff. 
- When you say, "We are interested in 5 countries" , you must say why because is what justify your further analysis. 
- "There's no data about population, which we need to see if the number of sightseeings has any relevance. 
We get it from google and add it to countries:
As it was discovered that the data only had a few countries with a similar high Income, no economic index will be analyzed. Analysis by states could be done later on if found income data for states." When you try to run some analysis with 2 different datasets, you need to be careful the way you set your hypothesis. You have ufosights temporal data set, you have imported a new data set of population to add to your countries. This only makes sense in case you get the population on the range date you want to study. Let's seat this afternoon in a break.